vizro-ai/docs/pages/user-guides/chart-examples.md

```diff
@@ -0,0 +1,95 @@
+# Explore VizroAI
+
+In this tutorial, we walk through the process of creating more complex plotly charts. The examples use data from the [plotly express data package](https://plotly.com/python-api-reference/generated/plotly.express.data.html).
+In the examples below we will use OpenAI `"gpt-4-0613"` model as we are going to request specific updates to the layout of the charts.
+
+### Create polar bar chart with Vizro-AI
+
+Polar Bar Charts are mostly used to visualize data that has both magnitude and direction, such as wind speed and direction at specific locations.
+They are particularly effective for displaying cyclical or directional data as they provide intuitive way of visualizing patterns in data across different directions.
+Let's create one using Vizro-AI.
+
+
+!!! example "Polar Bar Chart"
+
+    === "Code for the cell"
+        ```py
+        import vizro_ai
+        from vizro_ai import VizroAI
+        import plotly.express as px
+
+        from dotenv import load_dotenv
+        load_dotenv()
+
+        df = px.data.wind()
+
+        vizro_ai = VizroAI(model="gpt-4-0613")
+        vizro_ai.plot(df,
+                      """Describe wind frequency and direction using bar_polar chart.
+                         Increase the width and height of the figure.
+                         Improve layout by placing title to the left. Show legend""", explain=True)
+
+        ```
+    === "Resulting chart"
+        [![VizroAIChart1]][VizroAIChart1]
+
+    [VizroAIChart1]: ../../assets/user_guides/polar_bar_chart.png
```

Comment:
Is there a way how to output the same height across all the gifs/screenshots? Eventually we can set it here in the result panel? I just noticed that the surface_plot.gif is quite big compared to the others - is it zoomed in? 
If it's possible to set the same height on the results, that would be great!

Reply:
I don't know how to do it in MkDocs but I'd be tempted to resize them all in an image editor to the same width and upload them in that state.

Reply:
I'll try to resize them in snaggit. I'm not sure why its so large, the screengrab is actually smaller than the map chart one.

Reply:
That's a good idea! I just don't know if this messes up the aspect ratio, but it would be good if we could consolidate that so it looks a bit cleaner / consistent here 👍

Reply:
You should be able to retain the aspect ratio. I'd aim for 1200 px max width, or even 800 px.

Reply:
All looks good to me, thanks!